Server/controllers/controllerUtils.js

```
@@ -12,32 +12,15 @@ const handleError = (error, serviceName, method, status = 500) => {
-						socket.end();
-					}
-				}
-			);
-		} catch (error) {
-			reject(error);
-		}
-	});
+		const cert = await sslChecker(hostname);
+		return cert;
+	} catch (error) {
+		throw error;
```

Biome

[error] The catch clause that only rethrows the original error is redundant.

These unnecessary catch clauses can be confusing. It is recommended to remove them.

(lint/complexity/noUselessCatch)